.release-plan.json

```diff
@@ -10,103 +10,31 @@
       "oldVersion": "3.0.0"
```

Comment:
I'm confused by the diff in this file -- were its contents stale?

Reply:
if the contents are stale, should we clear the config for each package after its release?

Reply:
would also work better when publishing bugfixes in lower versions